src/app/header/header.component.scss

```diff
@@ -37,7 +39,7 @@
     .nav-mob-scroll{
         overflow-x: scroll !important;
         .nav-list-mob-scroll{
-            width: 150%;
+            width: 170%;
```

Comment:
on mobile view the secondary menu bar covers all of the space... I make it scrollable for time being and use the exact space